docs/overview.yaml

```
@@ -110,6 +110,7 @@ General algebra:
     splitting field: 'polynomial.splitting_field'
     perfect closure: 'perfect_closure'
     Galois correspondence: 'is_galois.intermediate_field_equiv_subgroup'
+    Abel-Ruffini theorem: 'solvable_by_rad.is_solvable' (one direction)
```

I think what comes after the colon has to be one declaration name
```suggestion
    Abel-Ruffini theorem (one direction): 'solvable_by_rad.is_solvable'
```